### PR TITLE
Changes to seeds.rb to fix run time errors 

### PR DESCRIPTION
**Pull request by RajeevHejib**
When executing 'bundle exec rake db:seeds' at the time of setting up an environment we encounter error.  The statement 'Feature.find(name: :volunteer_ops_create).activate' and the next three statements fail.  
```
ActiveRecord::RecordNotFound: Couldn't find Feature with 'id'={:name=>:volunteer_ops_create}
/Users/rajeevhejib/.rvm/gems/ruby-2.0.0-p643/gems/activerecord-4.2.3/lib/active_record/core.rb:155:in `find'
```
These lines have been changed to eliminate the error.  Here is a link to the original Pivotal  Tracker ticket: https://www.pivotaltracker.com/story/show/112492939

### CODE REVIEW

**Comment by tansaku** [comment]
@RajeevHejib looks good - maybe just add a description and a link to the pivotal ticket - also I might be wrong but the commit format might need to be [Fixes #112492939] but let's see ...

**Comment by tansaku** [comment]
thanks @RajeevHejib 

BTW, whenever talking about an error it's great to include it in markdown format like this:

```
ActiveRecord::RecordNotFound: Couldn't find Feature with 'id'={:name=>:volunteer_ops_create}
/Users/rajeevhejib/.rvm/gems/ruby-2.0.0-p643/gems/activerecord-4.2.3/lib/active_record/core.rb:155:in `find'
```

**Comment by tansaku** [comment]
thanks @RajeevHejib - could you perhaps rename the file in docs to README.md - and add a title section saying Feature Flags - and I'll add other sections so it becomes a general README on how we run the project - many thanks in advance

File: doc/Readme_For_Setting_Up_New_Features [hunk 1]
```diff
@@ -0,0 +1,6 @@
+When setting up new feature please make sure the feature is activated in seed.sb file.
```
Comment (tansaku) [inline doc/Readme_For_Setting_Up_New_Features]: typo - should be `seeds.rb` and I would make it specifically `db/seeds.rb` to help people to know where to find it.

We should also mention that they should create a migration, and link to relevant documentation on Feature flags, e.g. http://martinfowler.com/bliki/FeatureToggle.html

**Comment by RajeevHejib** [comment]
@tansaku not sure why cucumber test failing in CI..  Don't get any errors locally.  BTW have changed the  file to README.md and also spotted some typo so corrected those too.

**Comment by tansaku** [comment]
@RajeevHejib not your fault - we have some timesensitive javascript things now that are failing intermittently - I need to fix them - in the meantime I have restarted your build ...

**Comment by tansaku** [comment]
README looks good @RajeevHejib - I'd just rename the directory to `docs` plural ...

**Pull request merged**